addons/dragging/functions/fnc_canCarry.sqf

```diff
@@ -21,7 +21,11 @@ params ["_unit", "_target"];
 private _alive = alive _target;
 private _isPerson = _target isKindOf "CAManBase";
 
-if !((_alive || _isPerson) && {_target getVariable [QGVAR(canCarry), false]} && {isNull objectParent _target}) exitWith {false};
+if !(
+    (_alive || _isPerson)
+    && {_target getVariable [QGVAR(canCarry), false]}
+    && {!_isPerson || {isNull objectParent _target}}
```

Comment:
The code below would address the problem by adding a minimal amount of possible cases. I'm a bit hesitant about going for `!_isPerson`, because in theory it would allow for non-person objects that have an `objectParent` that isn't a corpse to be dragged. In practice I'm not sure if that's even possible, so idk if I'm worrying over nothing.

```suggestion
    && {isNull objectParent _target || {!isNull getCorpse _target}}
```